test/classes/errors/newValueNotType2.good

```diff
@@ -1,6 +1,2 @@
-newValueNotType2.chpl:7: error: cannot use decorator borrowed on a value
-newValueNotType2.chpl:7: error: duplicate decorators - borrowed owned C
-newValueNotType2.chpl:7: error: invalid use of 'new'
-newValueNotType2.chpl:10: error: cannot use decorator unmanaged on a value
-newValueNotType2.chpl:10: error: duplicate decorators - unmanaged owned C
-newValueNotType2.chpl:10: error: invalid use of 'new'
+newValueNotType2.chpl:7: error: type in 'new' expression is missing its argument list
+newValueNotType2.chpl:10: error: type in 'new' expression is missing its argument list
```

Comment:
Ugh this is unfortunate. I am OK with it nonetheless.

Reply:
I think this is the right error for this case. If the parens are added, it gives an error about not having a type, e.g.

```
newValueNotType4.chpl:9: error: 'new' must be followed by a type expression
```

I will add a sibling newValueNotType4.chpl test with the parens to make sure we notice if this changes.